### PR TITLE
ignore everything exceot yaml, json, yml, jsonld

### PR DESCRIPTION
**Pull request by OlegZharkov**
So we don't have this situation again
https://github.com/bio-tools/content/pull/474/files#diff-c00ed658f81e670cc476b2a23411947527d491f1a55ff14cc58fd33ee30760d3

### CODE REVIEW

**Comment by bgruening** [comment]
@hmenager is that ok with you?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
-*.idea
-venv
+*
+
+!*.json
```
Comment (bgruening) [inline .gitignore:3]: can we restrict this to the data folder and those file endings? We probably want to change python scripts and so on.

Comment (OlegZharkov) [reply .gitignore:3]: right, makes sense

**Review by hmenager — APPROVED** [review]
Great catch, thanks @OlegZharkov @bgruening !

**Pull request merged**